keylime-agent/src/main.rs

```diff
@@ -993,7 +998,11 @@ mod testing {
                 .join("test-rsa.pem");
 
             let (nk_pub, nk_priv) =
-                crypto::testing::rsa_import_pair(rsa_key_path)?;
+                crypto::testing::rsa_import_pair(rsa_key_path.clone())?;
+
+            // Generate separate ephemeral payload keys for testing
```

Comment:
I would extend this comment, and would also evaluate adding a debug! comment for this:
```bash
// Generate ephemeral RSA key pair for the payload mechanism. This key is not
// persisted and is unique to this agent's runtime instance.
debug!("Generating ephemeral RSA key pair for payload mechanism");
```